File: jinwoo/2주차/완전 제곱수.py
import math
    
M=int(input())
N=int(input())

sum=0
minIdx=float("inf")

for i in range(1,101):
    if i*i>=M and i*i<=N:
        sum+=i*i
        minIdx = min(minIdx,i)

if sum==0:
    print(-1)
else :
    print(sum)
    print(minIdx*minIdx)

File: jHyuk/week2/완전제곱수.cpp
#include <iostream>
#include <vector>
#include <algorithm>
#include <math.h>

#define INF 2147483647

using namespace std;

int main() {
	int M, N, sum = 0, min = INF;

	cin >> M >> N;

	for (int i = M; i <= N; i++) {
		if ((int)sqrt(i) - sqrt(i) == 0) {	//제곱근이 정수이면 반환
			if (min > i) min = i;
			sum += i;
		}
	}

	if (min != INF) cout << sum << endl << min << endl;
	else cout << -1;

	return 0;
}
